.buildkite/pipeline.deploy-stage.yml

```diff
@@ -1,50 +1,36 @@
 steps:
   - label: "Deploy (catalogue stage)"
     branches: "main"
+    command: |
+      ENV_TAG="env.staging" .buildkite/scripts/update_ecr_image_tag.sh \
+        uk.ac.wellcome/concepts
+        uk.ac.wellcome/items
+        uk.ac.wellcome/search
+
+      CLUSTER="catalogue-api-2021-04-26" .buildkite/scripts/redeploy_ecs_services.sh \
+        stage-concepts-api \
+        stage-items-api \
+        stage-search-api
```

Comment:
The repetition is a bit annoying here, and we could maybe collapse these into a single pipeline, but this should work and I don't want to make too many more changes right now.